src/main/java/com/google/sps/servlets/NotificationServlet.java

```diff
@@ -192,4 +193,26 @@ private void createNotification(String query, String notificationUrl, String not
       logger.log(Level.SEVERE, ex.getMessage(), ex);
     }
   }
+
+  /**
+   * Fetches ID of question that is related to the answer that was commented.
+   */
+  private int getIdOfAnsweredQuestion(int answerId) {
+    String query = "SELECT question_id FROM Answer WHERE id = " + answerId;
+    int answeredQuestionId = 0;
```

Comment:
Are you database id's 0 indexed? Is there a question that actually does have the ID of 0? If so then it might be better to use -1 instead.

Reply:
Both questions and answers start in 1 and increment automatically.

Reply:
But I still changed it to -1 because I guess that 1 is the standard.